src/ws.js

```diff
@@ -88,17 +89,19 @@ class User {
  * called several times, until some content exists. So you need to handle concurrent calls.
  * @param {(ws:uws.WebSocket<User>)=>void} [conf.openWsCallback] - called when a websocket connection is opened
  * @param {(ws:uws.WebSocket<User>,code:number,message:ArrayBuffer)=>void} [conf.closeWsCallback] - called when a websocket connection is closed
+ * @param {Redis | undefined} redis
  */
 export const registerYWebsocketServer = async (
   app, 
   pattern, 
   store, 
   checkAuth, 
-  { redisPrefix = 'y', initDocCallback = () => {}, openWsCallback = () => {}, closeWsCallback = () => {} } = {}
+  { redisPrefix = 'y', initDocCallback = () => {}, openWsCallback = () => {}, closeWsCallback = () => {} } = {},
+  redis
```

Comment:
Ich würd redis zum conf-Objekt-Parameter hinzufügen, weil da alle optionalen Parameter gesammelt sind. Und ich fänd nen Namen wie "redisInstance" verständlicher.

Reply:
ja redisInstance ist gut, und in options können wir auch rein packen

Reply:
Ich habs umbenannt, aber in "ioRedisInstance", weil das den Bibliotheken-Wechsel deutlich macht, denn darum gehts ja.